### PR TITLE
Remove installer issue and update FWLink to point to the updated inst…

### PR DESCRIPTION
**Pull request by leecow**

### CODE REVIEW

File: docs/core/windows-prerequisites.md [hunk 1]
```diff
@@ -61,14 +61,6 @@ NuGet is the package manager for the Microsoft development platform including .N
 
 ## .NET Core Tools for Visual Studio 2015
 
-Download and install the [.NET Core Tooling Preview 2 for Visual Studio 2015](https://go.microsoft.com/fwlink/?LinkId=817245). 
+Download and install the [.NET Core Tooling Preview 2 for Visual Studio 2015](http://go.microsoft.com/fwlink/?LinkID=824849).
```
Comment (mairaw) [inline docs/core/windows-prerequisites.md]: why did you remove the https?

Comment (leecow) [inline docs/core/windows-prerequisites.md]: Oversight, copy/paste from the fwlink tool. Have updated the pr to https.

**Comment by mairaw** [comment]
LGTM!
